### PR TITLE
Update labels-projects.md

### PR DESCRIPTION
**Pull request by Youssef1313**
Not sure if this would be the ideal change.

Related to https://github.com/dotnet/docs/pull/21574.

cc: @adegeo

### CODE REVIEW

**Comment by Jim-Parker** [comment]
@Youssef1313 can we just update the link instead of adding the note about it being moved?
cc @adegeo

**Comment by adegeo** [comment]
I would just delete the entry.

File: Contribute/dotnet/labels-projects.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Each .NET Guide is noted with the `:books: Area` prefix and has a dark blue back
 - [API Reference](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20API%20Reference)
 - [Azure .NET SDK](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20Azure%20.NET%20SDk)
 - [C# Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20C%23%20Guide)
-- [Desktop Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20Desktop%20Guide)
+- [Desktop Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20Desktop%20Guide) - moved to [dotnet/docs-desktop](https://github.com/dotnet/docs-desktop/issues) repository.
```
Comment (Youssef1313) [inline Contribute/dotnet/labels-projects.md:56]:
```suggestion
```

**Review by adegeo — APPROVED** [review]

**Comment by Jim-Parker** [comment]
@Youssef1313 is this ready to merge now?

**Comment by Youssef1313** [comment]
@Jim-Parker Yes you can merge.

**Review by BillWagner — APPROVED** [review]
This LGTM @Youssef1313 

@Jim-Parker You can :shipit: when you are ready.

**Pull request merged**